evalml/pipelines/utils.py

```diff
@@ -65,7 +65,7 @@
 from evalml.utils import get_time_index, infer_feature_types
 from evalml.utils.gen_utils import contains_all_ts_parameters
 
-_UNSUPPORTED_FREQUENCIES_STL_DECOMPOSER = ["A", "T"]
+_UNSUPPORTED_FREQUENCIES_STL_DECOMPOSER = ["A", "Y", "T", "AS", "YS"]
```

Comment:
we might want to move this either 1.) into the stl_decomposer module and/or 2.) make it returnable by the decomposer itself.  Then we can dynamically know what decomposers support or do not.  You could also build the parametrization list and parametrize across STL decomposers in your decomposer test that way.

Part of the motivation behind the move, in my mind, is the answer to "where should I go to find what frequencies are not supported by X Decomposer?"

Reply:
A lovely idea - filed #3795 to track this!